pykrige/ok.py

```diff
@@ -189,7 +189,7 @@ class OrdinaryKriging:
     def __init__(self, x, y, z, variogram_model='linear', variogram_parameters=None,
                  variogram_function=None, nlags=6, weight=False, anisotropy_scaling=1.0,
                  anisotropy_angle=0.0, verbose=False, enable_plotting=False,
-                 enable_statistics=False):
+                 enable_statistics=False, coordinates_type='euclidean'):

```

Comment:
Just realized I did this already in line 228/229. Moved the definition of self.coordinates_type behind the check so that it will always be valid if set.